x-pack/platform/plugins/shared/fleet/server/services/agents/auto_upgrade_agents_status.ts

```diff
@@ -26,7 +27,7 @@ export async function getAutoUpgradeAgentsStatus(
     .listAgents({
       showInactive: false,
       perPage: 0,
-      kuery: `${AGENTS_PREFIX}.policy_id:"${agentPolicyId}"`,
+      kuery: `${AgentStatusKueryHelper.buildKueryForActiveAgents()} AND ${AGENTS_PREFIX}.policy_id:"${agentPolicyId}"`,
```

Comment:
would it make sense to add some additional unit tests for these queries? I'm afraid they're easy to break without it

Reply:
added unit test